### ISSUE TITLE
Reset bookmarks

### ISSUE THREAD

**Issue by chchen-stanford** [holzschu/a-shell#283]
Is it possible to have a command to quickly reset all bookmarks to factory state? I have some one-off bookmarks allowing temporary access to a-Shell. At the moment, I use `delete mark` to clean up things, but it is very tedious on a smartphone.

**Comment by holzschu** [comment]
Good idea. I didn't think about it, but it's relatively easy. `deletemarks --all` will probably be the syntax.

**Comment by holzschu** [comment]
This feature is available in TestFlight build 180 (soon to become version 1.7.4 on the AppStore). We only keep home, shortcuts, group, cloud and iCloud.
